### ISSUE TITLE
Problem on 1.7.5.0-beta Property CookieClass->name is empty.

### ISSUE THREAD

**Issue by rdy4ever** [GuillaumeKadolis/k_cookies#5]
Hi 
Just tested your module on 1.7.5.0-beta and, altough it installs fine, when I try to eneble any service I get this error. It could be related to: https://devdocs.prestashop.com/1.7/modules/core_updates/1.7.5/

```
[PrestaShopException]
Property CookieClass->name is empty.
at line 984 in file classes/ObjectModel.php

979.             }
980. 
981.             $message = $this->validateField($field, $this->$field);
982.             if ($message !== true) {
983.                 if ($die) {
984.                     throw new PrestaShopException($message);
985.                 }
986. 
987.                 return $error_return ? $message : false;
988.             }
989.         }
ObjectModelCore->validateFields - [line 284 - classes/ObjectModel.php]
ObjectModelCore->getFields - [line 551 - classes/ObjectModel.php]
ObjectModelCore->add - [line 508 - classes/ObjectModel.php] - [2 Arguments]
ObjectModelCore->save - [line 418 - modules/k_cookies/k_cookies.php]
k_Cookies->_postProcess - [line 209 - modules/k_cookies/k_cookies.php]
k_Cookies->getContent - [line 905 - controllers/admin/AdminModulesController.php]
AdminModulesControllerCore->postProcessCallback - [line 1165 - controllers/admin/AdminModulesController.php]
AdminModulesControllerCore->postProcess - [line 270 - classes/controller/Controller.php]
ControllerCore->run - [line 509 - classes/Dispatcher.php]
DispatcherCore->dispatch - [line 99 - admin-dev/index.php]
```

**Comment by rdy4ever** [comment]
Hello. Any news on this one? 1.7.5 is just around the corner.
Thanks

**Comment by duGuillaume** [comment]
Hi,
Prestashop 1.7.5 out today.
I tried to reproduce your bug with this final version but i can't.
Could you try again, and tell me which service bug if you still have an error.

**Comment by loicy** [comment]
Hi Gullaume,

Any news ?

Btw, working great on localhost @ prestashop 1.7.2

I cant install (didnt show on module after upload on ftp). Any suggest ?

Regards,

**Comment by duGuillaume** [comment]
@loicy Do you have errors when you install the module ?

**Comment by loicy** [comment]
@duGuillaume No error no, I do not even see it in the modules for activation ! Any idea ?

**Comment by duGuillaume** [comment]
I just tried with a fresh install on presta 1.7.5.1.
I uploaded the module by manage page on BO `?controller=AdminPsMboModule` the `install a module`
and everthing work fine.
try to go to module admin by this link `index.php?controller=AdminModules&configure=k_cookies`

if you have an error may be you had an install pb. Try again to install.
Here my zip (from master): 
[k_cookies.zip](https://github.com/duGuillaume/k_cookies/files/2876976/k_cookies.zip)
